solana/program_v2/programs/gateway_v2/src/state/gatekeeper.rs

```diff
@@ -209,7 +209,7 @@ pub enum GatekeeperState {
 }
 
 impl OnChainSize for GatekeeperState {
-    const ON_CHAIN_SIZE: usize = 1;
+    const ON_CHAIN_SIZE: usize = 1 + 2;
```

Comment:
This is meant to be `1 + largest variant size` https://book.anchor-lang.com/anchor_references/space.html

I am not sure what this should be 😅